js/sia.js

```diff
@@ -3,20 +3,19 @@
 // Library for making requests
 const Request = require('request')
 
-// Necessary node libraries
-const Path = require('path')
+// Necessary node libraries to make sia.js emit events
 const Util = require('util')
 const EventEmitter = require('events')
 
 /**
- * DaemonManager, a closure, initializes siad as a background process and
+ * Sia, a closure, initializes siad as a background process and
  * provides functions to interact with it
- * @class DaemonManager
+ * @class Sia
  */
-function DaemonManager () {
+function Sia () {
```

Comment:
it's not really Sia either? It's supposed to wrap siad right? I am really tired, I might be missing something.